test/test_irods_http_api.py

```diff
@@ -2113,6 +2113,55 @@ def test_server_reports_error_when_op_is_not_supported(self):
     def test_return_error_on_missing_parameters(self):
         pass
 
+    def test_server_correctly_handles_embedded_CRLFs_in_multipart_form_data(self):
```

Comment:
issue number in the test name?

Reply:
We decided not to do that at this time a while back.

I think it was because the project is still considered experimental and hasn't reached 1.0 status.

Reply:
oh?  but... hrm.   okay.